docker-compose.yml

```diff
@@ -1,39 +1,37 @@
-version: "2"
+version: "3"
 services:
 
   db:
     image: postgres
     ports:
       - "5432:5432"
 
-  rabbitmq:
-    image: rabbitmq
-    hostname: rabbitmq
+  sqs:
+    image: pakohan/elasticmq
+    hostname: sqs
     ports:
-      - "5672:5672"
-      - "15672:15672"  # here, we can access rabbitmq management plugin
+      - 9324:9324
 
   django:
     container_name: ngx_django
     hostname: django
     env_file:
-      - docker/dev.env
+      - docker/dev/docker.env
     build:
       context: ./
       dockerfile: docker/dev/django/Dockerfile
     ports:
       - "8000:8000"
-    command: ["./docker/wait-for-it.sh", "db:5432", "--", "sh", "/code/docker/dev/django/container-start.sh"]
     depends_on:
       - "db"
+      - "sqs"
 
-  submission-worker:
+  worker:
     env_file:
-      - docker/dev.env
+      - docker/dev/docker.env
     build:
       context: ./
-      dockerfile: docker/dev/rabbitmq/Dockerfile
-    command: ["./docker/wait-for-it.sh", "django:8000", "--", "python", "scripts/workers/submission_worker.py"]
+      dockerfile: docker/dev/worker/Dockerfile
     depends_on:
       - "django"
 
```

Comment:
I think this service name must be angular-app rather than angularjs.

Reply:
Let's change for the container name.